Source/santa-driver/SantaCache.h

```diff
@@ -206,7 +206,7 @@ template<typename KeyT, typename ValueT> class SantaCache {
       lock(bucket);
       struct entry *entry = (struct entry *)((uintptr_t)bucket->head - 1);
       while (entry != nullptr) {
-        if (entry->key != 0) ++count;
+        if (entry->key != key_zero_) ++count;
```

Comment:
> I don't think there's anything special about entry->key being zero is there?

No, the counting code was wrong and was checking for a zero key rather than a zero value, which is wrong.